docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -62,43 +62,122 @@ and details on the intrapartum and neonatal time periods.
 3.0 Concept model and submodels
```

Comment:
Maybe this is just my pet peeve, but isn't this whole _document_ the concept model, so this section should be titled "Concept model diagram" or something?